tests/ir_lowering/unsupported_variants.ll

```diff
@@ -12,7 +12,6 @@
 ;         br bb2
 ;      bb2:
 ;         unimplemented <<  %{{13}} = fadd nnan float %{{3}}, %{{3}}>>
-;         unimplemented <<  %{{14}} = udiv exact i32 %{{2}}, 1>>
```

Comment:
How comes we removed this?

Reply:
Because we now ignore `exact` for all binary operations.

Reply:
ah, no i get it. we are safe to ignore all exacts, not just for ashr.

Reply:
Sorry, we did discuss this, just it slipped the mind :)